src/automations/JobPost.ts

```diff
@@ -197,7 +201,31 @@ export default class JobPost {
             question.answer_type_key = question.answer_type.key;
             delete question.answer_type;
             question.id = null;
+            if (
+                question["question_options"] &&
+                question["question_options"].length > 0
+            ) {
+                const options = question["question_options"]
+                    .map((question: { label: string }) => question.label)
+                    .reduce((str1: string, str2: string) => str1 + "\n" + str2);
+                delete Object.assign(question, {
+                    question_options_text: options,
+                })["question_options"];
+            }
```

Comment:
I find this confusing. Are we deleting and adding a property at the same time?

Reply:
The api that creates a job post requires specific fields. For example we have "question_options" field but API requires "question_options_text". We need to set "question_options_text" field and delete the  "question_options" field. In the code when we need to do that we used this convention. The first parameter of the "assign" is the object that is needed to be updated, the second parameter is the object that is going to be assigned. This will return the updated object. Then "delete" will delete the field in this case it is  "question_options". It can be written in two lines first assign then delete, but I find it cleaner.